src/wavesurfer.js

```diff
@@ -92,11 +92,6 @@ var WaveSurfer = {
         this.drawer = Object.create(WaveSurfer.Drawer[this.params.renderer]);
         this.drawer.init(this.container, this.params);
 
-        this.drawer.on('redraw', function () {
-            my.drawBuffer();
```

Comment:
As noted in the commit message, I could not find any place where `redraw` is fired on drawer:

> The `redraw` event is never fired on the drawer (only the wavesurfer
> object itself). The code has been unused since [resize handling has
> been removed from the drawer][1].
>
> [1]: https://github.com/katspaugh/wavesurfer.js/commit/50a89e477a68792c522672a0c693e5456e7ab672#diff-0333441833ed71523c3686dac5804d80L145